standalone/src/models/order/parcel/builder.spec.ts

```diff
@@ -68,6 +68,8 @@ describe('builder', () => {
       .id(id)
       .key(key)
       .measurements(measurements)
+      // Parcel will be migrated and this file won't exist anymore
+      // @ts-expect-error - TrackingData is not a valid type for Parcel
```

Comment:
The idea is to migrate `Parcel`, for that I need to migrate `TrackingData`, `DeliveryItem` and `ParcelMeasurements`. 

This file will be removed shortly